gems/pending/spec/appliance_console/database_replication_standby_spec.rb

```diff
@@ -208,4 +215,85 @@
       expect(subject.start_postgres).to be_truthy
     end
   end
+
+  context "#configure_repmgrd" do
+    it "returns true if not setup to do the configuration" do
+      subject.run_repmgrd_configuration = false
+      expect(subject.configure_repmgrd).to be true
+    end
+  end
+
+  context "#write_repmgrd_config" do
+    let(:config_path) { Tempfile.new("repmgr_config").path }
+
+    before do
+      stub_const("ApplianceConsole::DatabaseReplication::REPMGR_CONFIG", config_path)
+    end
+
+    after do
+      FileUtils.rm_f(config_path)
+    end
+
+    it "appends the correct data to the config file" do
+      File.write(config_path, "some other stuff here\n")
+
+      subject.write_repmgrd_config
+
+      expect(File.read(config_path)).to eq(<<-EOS.gsub(/^\s+/, ""))
```

Comment:
Ugh...I know you *really* want to use the squiggly heredoc.  Can you use active_support's strip_heredoc method?